<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormNavigationQuery</class>
 <widget class="QWidget" name="FormNavigationQuery">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
   <item>
    <widget class="QPushButton" name="insertButton">
     <property name="text">
      <string>新增</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="deleteButton">
     <property name="text">
      <string>删除</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="submitButton">
     <property name="text">
      <string>提交</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
